tests/lang_callable/iter/titer2.nim

```diff
@@ -8,9 +8,13 @@ discard """
 2
 3
 4'''
-  cmd: "nim $target --gc:none --hints:on --warnings:off $options $file"
+  #cmd: "nim $target --gc:none --hints:on --warnings:off $options $file"
```

Comment:
```suggestion
  #matrix: "--gc:none --hints:on --warnings:off"
```

Just to phasing out more of these `cmd`s; doesn't have to be done figured while the line is being edited regardlesss.